lib/utilities/execa.js

```diff
@@ -1,20 +0,0 @@
-'use strict';
```

Comment:
This was not used consistently, and we can get the same or more info by setting `NODE_DEBUG=execa` when needed.